recipe/build.sh

```diff
@@ -1,5 +1,25 @@
-set -ex
+set -o xtrace -o nounset -o pipefail -o errexit
 
-cargo install --path . --root ${PREFIX}
+# Required for cross-compiling with pkg-config
+export PKG_CONFIG_SYSROOT_DIR=$PREFIX
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+export CARGO_BUILD_RUSTFLAGS="$CARGO_BUILD_RUSTFLAGS -L all=$PREFIX/lib"
+
+if [[ "$c_compiler" == "clang" ]]; then
+  echo "-L$BUILD_PREFIX/lib -Wl,-rpath,$BUILD_PREFIX/lib" > $BUILD_PREFIX/bin/$BUILD.cfg
```

Comment:
can we add all these configuration to rust itself, then. they will be available for all?